### DISCUSSION TITLE
part-ii/2015-7/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#134] [Question comments]
Part II 2015 Algebraic Topology: Paper 1, Section II, H 

https://questions.tripos.org/part-ii/2015-7/

**Comment by ArthurConmy** [comment]
Bookwork and then the group has presentation $G = \langle b, c | bc^2 = c^2b \rangle$. 

My idea for showing that this is infinite and non-abelian is to show that each $g \in G$ has unique representation (<b>I'd love a comment if there's a different way to see that the group is infinite, this feels a little but handwavy</b>) as a finite string

$$ c^{p_1} b^{p_2} c^{p_3} ... $$

subject to the constraints that $p_1, p_2, ... p_n \neq 0$ and more importantly that $p_3, p_5, p_7, ... \in \{ -1 , +1 \}$. 

Then this is clearly infinite and moreover non-abelian since for example $cbc^{-1}$ and $c^2$ do not commute: we have unique representations

$$ (cbc^{-1}) c^2 = cbc \neq c^3 b c^{-1} = (c^2)(cbc^{-1}) $$.

**Reply by asheshbati** [reply]
That example for non-commuting elements is very slick.

To show that the group is infinite, we can construct a surjective homomorphism to an infinite group. In particular, we can have a surjective map $\Phi: \langle{b, c}\rangle \rightarrow D_{\infty}$ Where we define $D_{\infty}$ as the group generated by integer translations and reflection across 0, on $\mathbb{R}$. Noting that this respects the group relations (either permutation of sending $b,c$ to $r,s$ works here), $\Phi$ factors through the quotient group $G$ and hence we get the surjective homomorphism desired.
